bodhi/server/consumers/signed.py

```diff
@@ -0,0 +1,88 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+The "signed handler".
+
+This module is responsible for marking builds as "signed" when they get moved
+from the pending-signing to pending-updates-testing tag by RoboSignatory.
+"""
+
+import pprint
+
+import fedmsg.consumers
+
+from pyramid.paster import get_appsettings
+from sqlalchemy import engine_from_config
+
+from bodhi.server.util import transactional_session_maker
+from bodhi.server.models import Base, Build, Release
+
+import logging
+log = logging.getLogger('bodhi')
```

Comment:
For [PEP-8](http://pep8.org/#imports), let's regroup these imports:

```
import logging
import pprint

import fedmsg.consumers
from pyramid.paster import get_appsettings
from sqlalchemy import engine_from_config

from bodhi.server.util import transactional_session_maker
from bodhi.server.models import Base, Build, Release
```

That way I don't have to fix that when I get around to adding this new module to flake8's automatic test ☺